app/src/main/java/com/vultisig/wallet/data/repositories/ChainsOrderRepository.kt

```diff
@@ -0,0 +1,107 @@
+package com.vultisig.wallet.data.repositories
+
+import com.vultisig.wallet.data.db.ChainOrderDao
+import com.vultisig.wallet.data.db.models.ChainOrderEntity
+import kotlinx.coroutines.Dispatchers.IO
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.withContext
+import javax.inject.Inject
+
+
+internal interface ChainsOrderRepository {
+    fun loadByOrders(): Flow<List<ChainOrderEntity>>
+    suspend fun updateItemOrder(lowerVal: String?, middleVal: String, upperVal: String?)
+    suspend fun delete(name: String)
+    suspend fun find(name: String): ChainOrderEntity
```

Comment:
I can't find any usage of this method, and I can't find any public usage of findMaxOrder. Consider removing them or making them private in implementation.